controllers/git_tekton_resources_renovater_test.go

```diff
@@ -61,11 +62,12 @@ var _ = Describe("Git tekton resources renovater", func() {
 				repositories := generateRepositories(installedRepositoryUrls)
 				return []github.ApplicationInstallation{generateInstallation(repositories)}, "slug", nil
 			}
-			componentNamespacedName := createComponentForPaCBuild(getComponentData(componentConfig{gitURL: "https://github/test/repo3"}))
+			componentNamespacedName := createComponentForPaCBuild(getComponentData(componentConfig{componentKey: types.NamespacedName{Name: "testnotrigger"}, gitURL: "https://github/test/repo3"}))
```

Comment:
What is the reason to change the component name?

Reply:
changed them back

Reply:
Which exactly? Can we do proper cleanup?